### PR TITLE
feat(subscribe): L3-3829 add ability to autofocus the input

### PR DESCRIPTION
**Pull request by scottdickerson**
**Jira ticket**

[Jira Ticket ID](https://phillipsauctions.atlassian.net/browse/L3-3829)

**Screenshots**
| Before | After |
| ------ | ----- |
| image | image |

**Summary**

Users would expect to be focused by default on the email input when the subscribe form opens, this change adds a property to be set to allow it.

**Change List (describe the changes made to the files)**

- change(*.scss): fix input component focus states to match others (outline black)
- change(Subscribe): add autoFocus prop that focuses on the Input when set

**Acceptance Test (how to verify the PR)**

- Popup the Drawer and Subscribe stories and verify that the text input field is focused
- Verify when you close the Drawer, whichever element on the page had focus previously now has had it reset to them.

**Regression Test**

- Interact with the Input, Select, Dropdown components still look right when interacting with them with both mouse and keyboard.

**Evidence of testing**

image

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

### CODE REVIEW

**Review by constantinehuzenko — APPROVED** [review]
LGTM

File: src/patterns/Subscribe/Subscribe.tsx [hunk 1]
```diff
@@ -72,6 +72,7 @@ export interface SubscribeProps extends React.HTMLAttributes<HTMLFormElement> {
  */
 
 const Subscribe = ({
+  autoFocus = true,
```
Comment (sgrund14) [inline src/patterns/Subscribe/Subscribe.tsx:75]: would this ever get consumed somewhere like the footer?

might be weird to autofocus an element that's off screen unintentionally, if this is defaulting to true

Comment (scottdickerson) [reply src/patterns/Subscribe/Subscribe.tsx:75]: We're only planning to show it in some kind of Drawer at this point so I think this is a good default.

**Pull request merged**